src/test/spec/unified_runner/entity.rs

```diff
@@ -1,15 +1,29 @@
-use std::ops::Deref;
+use std::{fmt, ops::Deref};
 
 use crate::{bson::Bson, test::util::EventClient, Collection, Database};
 
+#[derive(Clone)]
 pub enum Entity {
     Client(ClientEntity),
     Database(Database),
     Collection(Collection),
-    Result(Bson),
+    Bson(Bson),
     None,
 }
 
+impl fmt::Debug for Entity {
```

Comment:
Why not use the derived `Debug` impl?

Reply:
I think this is from an old commit range; I deleted this in the most recent commit

Reply:
Whoops, my bad.